app/views/practices/show/implementation/_resource_link_attachment.html.erb

```diff
@@ -8,7 +8,15 @@
         <% end %>
         <% la.name = la.link_url unless la.name.present? %>
         <div class="margin-top-1 width-full line-height-26">
-          <li class="small-disc"><%= la.description %> <a class="usa-link <%= la.link_url.downcase().include?('marketplace.gov') ? '' : 'usa-link--external' %>" href="<%= la.link_url %>" target="_blank"><%= la.name || la.link_url %></a></li>
+          <li class="small-disc">
+            <%= la.description %>
+            <a class="usa-link <%= la.link_url.downcase().include?('marketplace.gov') ? '' : 'usa-link--external' %>"
```

Comment:
This is a repeat of a previously dismissed warning. The warning supposes we are checking for a malicious url when in actuality we are just determining the css class to attribute to a supplied email address. We trust our users to not introduce malicious urls.

Reply:
How about updating this `.erb` to use `#set_link_classes` on the `la.link_url`? It'll be nice to reuse some of the methods we've been using over in PageBuilder.

Reply:
Yeah, I think you're right re: updating the original method to be safer 👍

Reply:
@camillevilla On second thought, I think there is still no way for us to actually prevent any malicious links. We want users to be able to supply an external link and I don't think there's any way of actually preventing a malicious url. 

Sorry I got a bit confused from context switching, but basically CodeQl thinks we are trying to prevent external links but in actuality we are just trying to identify internal links for slightly different CS